components/director/internal/repo/get.go

```diff
@@ -39,7 +41,7 @@ func (g *SingleGetter) Get(ctx context.Context, tenant string, conditions Condit
 	err = persist.Get(dest, q, allArgs...)
 	switch {
 	case err == sql.ErrNoRows:
-		return &notFoundError{}
+		return apperrors.NewNotFoundError(conditions[0].Val)
```

Comment:
It would be better to have two constructors per error:
-  NewNotFoundError() +  NewNotUniqueError()
- NewNotFoundErrorWithID(identifier) / NewNotUniqueErrorWithID(identifier) <- but it has a better name 😛

Reply:
@aszecowka maybe you have a better idea for constructor name 😄